internal/pkg/cli/env_show_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: we can remove the year now :)
```suggestion
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```